README.rst

```diff
@@ -27,21 +27,23 @@ PennyLane Forest Plugin
 
 .. header-start-inclusion-marker-do-not-remove
 
-Contains the PennyLane Forest plugin. This plugin allows three Rigetti devices to work with
+Contains the PennyLane Forest plugin. This plugin allows different Rigetti-pyQuil devices to work with
 PennyLane --- the wavefunction simulator, the Quantum Virtual Machine (QVM), and Quantum Processing
 Units (QPUs).
 
 `pyQuil <https://pyquil.readthedocs.io>`_ is a Python library for quantum programming using the
 quantum instruction language (Quil) --- resulting quantum programs can be executed using the
-`Rigetti Forest SDK <https://www.PennyLaneAI.com/forest>`_ and the `Rigetti QCS
-<https://www.PennyLaneAI.com/qcs>`_.
+`Rigetti Forest SDK <https://pyquil-docs.rigetti.com/en/stable/>`_ and the `Rigetti QCS
+<https://qcs.rigetti.com/>`_.
 
-`PennyLane <https://pennylane.ai>`_ is a machine learning library for optimization and automatic
-differentiation of hybrid quantum-classical computations.
+`PennyLane <https://pennylane.readthedocs.io>`__ is a cross-platform Python library for quantum machine
+learning, automatic differentiation, and optimization of hybrid quantum-classical computations.
```

Comment:
We have an updated description on the main site now:

> PennyLane is a cross-platform Python library for differentiable programming of quantum computers. Train a quantum computer the same way as a neural network.

Maybe worth adjusting this to include "differentiable programming of quantum computers"?